setup.cfg

```diff
@@ -21,9 +21,6 @@ testserver       = pytest --addopts "tests/gordo/server"
 testutil         = pytest --addopts "tests/gordo/util"
 testworkflow     = pytest --addopts "tests/gordo/workflow"
 
-# Black formatting
-testformatting   = pytest --addopts "tests/test_formatting.py"
-
```

Comment:
black will be checked now on the CI step at first place, not to wait for the tests to end.